src/components/Executions/ExecutionDetails/ExecutionMetadataExtra.tsx

```diff
@@ -54,7 +55,10 @@ export const ExecutionMetadataExtra: React.FC<{
     },
     {
       label: ExecutionMetadataLabels.serviceAccount,
-      value: authRole?.kubernetesServiceAccount || ExecutionMetadataLabels.securityContextDefault,
+      value:
+        authRole?.kubernetesServiceAccount ||
```

Comment:
Just saw you comment in the task, we don't need to support for `authRole` any more as it's a deprecated field, so line 59 should be safely removed.

Reply:
Can you please also update line 53 to use `securityContext.IamRole` instead of `authRole`

Reply:
Also this lines could also now be safely removed: https://github.com/flyteorg/flyteconsole/blob/24bdaeefcf32c079d1aaeb54b28cc6bf7395b42f/src/models/Execution/api.ts#L125-L127
😇